kmcd-driver.md

```diff
@@ -52,47 +52,48 @@ Function Calls
     syntax MCDStep ::= "call" MCDStep
  // ---------------------------------
     rule <k> call AS:AuthStep ~> CONT => contract(AS) . auth ~> AS </k>
```

Comment:
This is wrong. Msg.sender IS updated by a contract call.

Reply:
reverted